cmd/composer-cli/projects/depsolve.go

```diff
@@ -21,14 +22,16 @@ var (
 		Short: "Show the dependencies of all of the listed projects",
 		Example: `  composer-cli projects depsolve tmux
   composer-cli projects depsolve tmux --json
-  composer-cli projects depsolve tmux --distro fedora-38`,
+  composer-cli projects depsolve tmux --distro fedora-38
+  composer-cli projects depsolve tmux --distro fedora-38 --arch aarch64`,
 		RunE: depsolve,
 		Args: cobra.MinimumNArgs(1),
 	}
 )
 
 func init() {
-	depsolveCmd.Flags().StringVarP(&distro, "distro", "", "", "Return results for distribution")
+	depsolveCmd.Flags().StringVarP(&distro, "distro", "", "", "Distribution")
+	depsolveCmd.Flags().StringVarP(&arch, "arch", "", "", "Architecture")
```

Comment:
same as for `distro`, here too

Reply:
Added a Long description with more details.